python/ray/tune/suggest/variant_generator.py

```diff
@@ -172,6 +198,39 @@ def _generate_variants(spec: Dict) -> Tuple[Dict, Dict]:
             yield resolved_vars, spec
 
 
+def get_preset_variants(spec: Dict, config: Dict):
+    """Get variants according to a spec, but first overwrite spec config
+    variables with values from `config`"""
```

Comment:
btw, generally, the first line of the docstring should fit on one line (no line wrap)

Reply:
In this case I ignored this as we're not documenting this internal function anywhere. It was just for internal reference. I'll still fix it